pySDC/implementations/problem_classes/AdvectionEquation_1D_FD.py

```diff
@@ -180,7 +180,12 @@ def u_exact(self, t):
         if self.params.freq >= 0:
             xvalues = np.array([i * self.dx for i in range(self.params.nvars)])
             me[:] = np.sin(np.pi * self.params.freq * (xvalues - self.params.c * t))
+        elif self.params.freq == -1:  # gaussian
+            xvalues = np.array([i * self.dx for i in range(self.params.nvars)])
+            sigma = 1e-1
```

Comment:
Please make this sigma a parameter accessible from the frontend. Set a default value, if you want to.